doc/api/console.md

```diff
@@ -103,70 +103,33 @@ The global `console` is a special `Console` whose output is sent to
 new Console(process.stdout, process.stderr);
 ```
 
-### console.assert(value[, message][, ...args])
+### console.assert(value[, ...message])
 <!-- YAML
 added: v0.1.101
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: The implementation is now spec compliant (see
+                 [web-api-assert][]) and does not throw anymore.
 -->
-* `value` {any}
-* `message` {any}
-* `...args` {any}
+* `value` {any} The value tested for being truthy.
+* `...message` {any} All arguments besides the first are used as error message.
```

Comment:
This feels a tiny bit confusing just because it says "all arguments besides the first." Is there a way we could reword?

Reply:
I am open for suggestions ;-)

Reply:
Maybe something like "All arguments besides `value` are used as error message."? Just to make it clear that first = `value` not the first argument within `message`.